### PR TITLE
Add /keys endpoint

### PR DESCRIPTION
**Pull request by dpgraham**
* Took the approach of simply creating the equivalent /actions events that /keys endpoint would translate to

### CODE REVIEW

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Keys.java [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+                    // Key up event
+                    Action keyUpAction = new Action();
+                    keyUpAction.setType(KEY_UP);
+                    keyUpAction.setValue(key);
+                    keyActions.add(keyUpAction);
+                }
+                inputSource.setActions(keyActions);
+
+                actions.setActions(Collections.singletonList(inputSource));
```
Comment (mykola-mokhnach) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Keys.java]: can it be changed to 
```
  new Actions()
      .setAdapter
      .setActions
      .perform
```

?

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Keys.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                // Create keyboard input source
+                InputSource inputSource = new InputSource();
+                inputSource.setType(KEY);
+                inputSource.setId("keyboard");
+
+                // Add a list of keyDown + keyUp actions for each key
+                List<Action> keyActions = new ArrayList<>();
+                for (final String key:params.getValue()) {
+                    // Key down event
+                    Action keyDownAction = new Action();
```
Comment (mykola-mokhnach) [inline espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Keys.java]: I assume action builder would be also more readable 

```
keyActions.add(new Action()
  .withType
  .withValue
);
```

File: espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Keys.java (hunk 1 above)
Comment (dpgraham) [reply espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/Keys.java]: Yeah that's a good idea.

**Comment by dpgraham** [comment]
Added the builder pattern.

**Pull request merged**